tests/test_grounding.py

```diff
@@ -51,7 +56,12 @@ def test_terminal_grounder(self):
         def mock_input(prompt):
             return next(mock_input_gen)
 
-        with mock.patch('__builtin__.raw_input', mock_input):
```

Comment:
Unfortunately, I can't get `six.input` to import (Python 3.5 64-bit with `six` installed), so that's a no-go, I guess. It was a good idea, though I think it's understandable enough?